grammars/css.cson

```diff
@@ -1913,7 +1913,8 @@
             'match': '''(?x)
               (-?(?!\\d)(?:[\\w-]|[^\\x00-\\x7F]|\\\\(?:[0-9a-fA-F]{1,6}|.))+)
               \\s*
-              (?=[~|^\\]$*=]|/\\*)
```

Comment:
About to eat, but changing `(?:[\\w-]...)` to an atomic group `(?>[\\w-]...)` works.  Didn't run specs though.